### ISSUE TITLE
Finalize FPGA kustomize

### ISSUE THREAD

**Issue by msivosuo** [intel/intel-device-plugins-for-kubernetes#318]
- [ ] Move FPGA kustomization out of “beta” and align with changes in #301.
- [ ] Update documentation
- [ ] Update the screencast to match kustomization based deployment
- [ ] Add kustomization for `opae-nlb-demo` deployments

**Comment by rojkov** [comment]
Finally this issue is unblocked. Beware of the new version of the fpga.intel.com API group. It's v2 now.

**Comment by askervin** [comment]
Task split: move configurable namespace to a new task: https://github.com/intel/intel-device-plugins-for-kubernetes/issues/386.
This task updates FPGA kustomize functionality on par with the first version and makes it production version (drop beta).
